src/twisted/web/test/test_flatten.py

```diff
@@ -706,15 +706,15 @@ def render(self, request: Optional[IRequest]) -> Flattenable:
                     Exception while flattening:
                       \\[<unrenderable>\\]
                       <unrenderable>
-                      .*
+                      <Deferred at .* current result: <twisted.python.failure.Failure builtins.RuntimeError: example>>
                       File ".*", line \\d*, in _flattenTree
-                        element = await element
-                    RuntimeError: example
+                        element = await element.*
```

Comment:
Maybe something like this ?

```suggestion
                        element = await element.*
                        RuntimeError: example
```

Reply:
That's exactly what I suggested.  See resolved conversation above.  I resolved the conversation because I thought the slightly less precise `assertIn` was fine.